model/src/main/kotlin/AdvisorRecord.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2020 Bosch.IO GmbH
+ * Copyright (C) 2020-2021 Bosch.IO GmbH
```

Comment:
Commit message: s/independent on the advisor/independent of the advisor/

Reply:
The parameter is now named `advisorIndex`. Fixed the commit message.